categorize_adjectives/keywords.py

```diff
@@ -0,0 +1,91 @@
+"""Categorize Adjectives by looking for keywords in definition"""
+import os
+from nltk.corpus import wordnet as wn
+
+def find_adjectives(adjectives, categories, keywords):
+"""For each adjective, finding similar adjectives and adding to respective categories"""
```

Comment:
Please update the comments according to this format (go/pystyle#function-docs) i.e add returns, args.
Also, please change the description to have first line as a summary followed by a blank line and then a detailed description.

Same for all methods.